sources/packages/forms/src/form-definitions/institutionprofile.json

```diff
@@ -1,12 +1,10 @@
 {
   "title": "Institution Profile",
-  "display": "form",
-  "type": "form",
   "name": "institutionprofile",
   "path": "institutionprofile",
-  "tags": [
-    "common"
-  ],
+  "type": "form",
+  "display": "form",
```

Comment:
please enable the formatter  prettier and eslint

Reply:
same for all form form definition

Reply:
Also please use the JSON formatter in vs code as default for all the *.json files. 
![image](https://user-images.githubusercontent.com/54600590/221717336-11ef84d7-57c0-4218-b120-9685785af1d5.png)